src/macaron/slsa_analyzer/git_service/gitlab.py

```diff
@@ -103,6 +105,10 @@ def clone_repo(self, clone_dir: str, url: str) -> None:
         To clone a GitLab repository with access token, we embed the access token in the https URL.
         See GitLab documentation: https://docs.gitlab.com/ee/gitlab-basics/start-using-git.html#clone-using-a-token.
 
+        If we clone using the https URL with the token embedded, this URL will be store as plain text in .git/config as
```

Comment:
```suggestion
        If we clone using the https URL with the token embedded, this URL will be stored as plain text in .git/config as
```

Reply:
Fixed in https://github.com/oracle/macaron/pull/354/commits/6678fe22d14923dfb62282973f68e093b5abd38b